src/annotator/test/sidebar-test.js

```diff
@@ -184,7 +184,7 @@ describe('Sidebar', () => {
 
   describe('toolbar buttons', () => {
     it('shows or hides sidebar when toolbar button is clicked', () => {
-      const sidebar = createSidebar({});
```

Comment:
I presume removing the empty objects here is just cleanup?

Reply:
Yes, cleanup